packages/riipen-ui/src/components/Toolbar.test.jsx

```diff
@@ -0,0 +1,77 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import Toolbar from "./Toolbar.jsx";
+
+describe("<Toolbar>", () => {
+  it("renders without errors", () => {
+    let error;
+
+    try {
+      mount(<Toolbar />);
+    } catch (e) {
+      error = e;
+    }
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const wrapper = mount(<Toolbar />);
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
```

Comment:
Just a shout out to @licarm , Imo and you have been doing the snapshot test slightly differently from PR to PR. Sometimes we put this as part of the default props tests, and sometimes we make it a stand alone test like this.

We should be consistent. My 2 cents is that I like it as a stand alone test like this.

Reply:
We'll switch to a stand alone snapshot test!